README.md

```diff
@@ -16,6 +16,35 @@ For details about configuration and deployment of RBD and CephFS CSI plugins, se
 
 For example usage of RBD and CephFS CSI plugins, see examples in `examples/`.
 
+## Support Matrix
+
+Here is the matrix that describes the features supported by the different
+versions of Ceph CSI driver
+
+|              |        Features             | CSI driver Version | Kubernetes Version   |
+|--------------|-----------------------------|--------------------|----------------------|
```

Comment:
This matrix needs rework from the perspective of stating what is basically provided to higher order features.
I would use language as in the [spec](https://github.com/container-storage-interface/spec/blob/master/spec.md#goals-in-mvp) for cleaner alignment to features.
Remove features that are not supported yet, most in the CephFS case, or list them outside the table, to keep the table mostly around what is possible.
My list would look as follows,
- Dynamic provisioning and de-provisioning of a volume
   - For FS add, "sharable or RWX volumes"
   - For RBD add, "single-consumer or RWO volumes"
- Creating and deleting a snapshot
- Provision volumes from a snapshot
- Block mode sharable or RWX volumes

Reply:
The initial version had RWO and RWX, but the reviewers wanted specifically the terms Cephfs and RDB so they can know which mode(RWX, RWO etc) is supported by Cephfs and RBD.

Reply:
Have addressed most of the suggestions. Kept the feature that we do not support yet, as it would give a complete picture. Also, retained the version 1.0.0 in the column as its going to happen in the near future.